spark/pyspark/arctern_pyspark/plot.py

```diff
@@ -0,0 +1,128 @@
+                line_arry[idx,1]=coor[1]
+                idx = idx + 1
+            if 'lines' not in dict_collect:
+                dict_collect['lines']=[]
+            dict_collect['lines'].append(line_arry)
+    elif geo_dict['type'] == 'LineString':
+        line_arry=np.zeros([len(geo_dict['coordinates']),2])
+        idx = 0
+        for coor in geo_dict['coordinates']:
+            line_arry[idx,0]=coor[0]
+            line_arry[idx,1]=coor[1]
+            idx = idx + 1
+        if 'lines' not in dict_collect:
+            dict_collect['lines']=[]
+        dict_collect['lines'].append(line_arry)
+    elif geo_dict['type']=='Point':
+        if 'points' not in dict_collect:
+            dict_collect['points']=[]
+        dict_collect['points'].append(geo_dict['coordinates'])
+    elif geo_dict['type']=='MultiPoint':
+        if 'points' not in dict_collect:
+            dict_collect['points']=[]
+        dict_collect['points'].extend(geo_dict['coordinates'])
+    else:
+        from osgeo import ogr
+        geo = ogr.CreateGeometryFromJson(json.dumps(geo_dict))
+        if geo.HasCurveGeometry():
+            geo = geo.GetLinearGeometry()
+            geo = geo.ExportToJson()
+            geo = json.loads(geo)
+            _flat_geoms(geo,dict_collect)
+        else:
+            raise RuntimeError(f"unsupported geometry: {geo_dict}")
+            
+def _plot_collection(ax, plot_collect):
+    if(len(plot_collect)==0):
+        return None
+    
+    try:
+        from descartes.patch import PolygonPatch
+    except ImportError:
+        raise ImportError(
+            "The descartes package is required for plotting polygons in geopandas. "
+            "You can install it using 'conda install -c conda-forge descartes' "
+        )
+    try:
+        import matplotlib
+        from matplotlib.collections import PatchCollection,LineCollection
+        from matplotlib.colors import is_color_like
+    except ImportError:
+        raise ImportError(
+            "The matplotlib package is required for plotting polygons in geopandas. "
+            "You can install it using 'conda install -c conda-forge descartes' "
+        )
+    
+        
+    if 'polygons' in plot_collect:
+        collection = PatchCollection([PolygonPatch(geo) for geo in plot_collect['polygons']])
+        ax.add_collection(collection, autolim=True)
+    if 'lines' in plot_collect:
+        collection = LineCollection(plot_collect['lines'])
+        ax.add_collection(collection, autolim=True)
+    if 'points' in plot_collect:
+        x = [p[0] for p in plot_collect['points']]
+        y = [p[1] for p in plot_collect['points']]
+        collection = ax.scatter(x,y)
+    ax.autoscale_view()            
+
+def _plot_spark_data_frame(ax, geoms):
+    import json
+    from pyspark.sql.functions import col
+    
+    if len(geoms.columns) != 1:
+        raise RuntimeError(f"geoms should has only one colums, input schema = {geoms.schema}")
```

Comment:
raise RuntimeError(f"geoms should has only one colums, input schema = {geoms.schema}")
=>
raise RuntimeError(f"The input param 'geoms' should have only one column. geoms schema = {geoms.schema}")